### PR TITLE
Add quadslotv2

### PR DESCRIPTION
**Pull request by iplaisier**
- The RNO-G Hpol converter (xfdtd, csv files) is added to antennapattern.py 
- The latest Vpol and Hpol are  updated in the RNO-G detector description

### CODE REVIEW

File: NuRadioReco/detector/antennapattern.py [hunk 1]
```diff
@@ -97,6 +98,106 @@ def get_group_delay(vector_effective_length, df):
...
+        path to gain file
+    path_phases:
+        path to phases file
+        
+    Returns:
+    ----------
+    all paramters of the file
+    """""
+
+    with open(path_gain, 'r', encoding='mac_roman') as fin:
```
Comment (cg-laser) [inline NuRadioReco/detector/antennapattern.py:117]: do you need to specify an enconding? It seems that this is mac specific.

**Review by cg-laser — CHANGES_REQUESTED** [review]
I compared the new XFDTD import/conversion function with the existing ARA reader. It seems that the file format is quite similar, maybe the two parse functions could be combined? But probably not that important. 

Comparing the calculation of the effective length I find an additional "4" in the denominator. It seems correct (comparing with A.13 of the NuRadioReco paper) if the input file is the realized gain. 

Actually I think there are more things wrong. The "n" used in A.13 is the index of refraction for which the realized gain was calculated, i.e. n=1 as these simulations were done for air. But what should change is the wavelength. 
`wavelength = c / np.array(ff)` should be replaced with `wavelength = c / np.array(ff)/n`
and H should just be
`H_theta = wavelength  * (50 / (4 * np.pi * Z_0)) ** 0.5 * gain_theta ** 0.5 * np.exp(1j * phase_theta)`

The function makes a few specific assumptions for rescaling with n, it would be nicer to have this as an parameter for the conversion function. 

Can you add the assumption about the antenna orientation also to the doc string/

**Comment by cg-laser** [comment]
link to NuRadioReco paper: https://doi.org/10.1140/epjc/s10052-019-6971-5

**Comment by anelles** [comment]
@iplaisier, please address the comments. Otherwise, this will be left behind when merging.

**Comment by anelles** [comment]
@cg-laser can you re-review this?

**Review by cg-laser — APPROVED** [review]
I pushed a commit that refactors the code a bit to make it clearer what the underlying assumptions are. @Ilse are these the changes that we discussed via slack and is the code that you used to make the latest revision of the Hpol antenna?

**Comment by iplaisier** [comment]
There was still a bug in the Hpol converter of the latest model i send you in calculating the phases (what i mentioned 2 weeks ago). This updated in this version. Furthermore it's the same and all the changes we discussed are included.

**Pull request merged**